AppCenterDataStorage/AppCenterDataStorage/Internal/Client/MSTokenExchange.h

```diff
@@ -4,6 +4,7 @@
 #import <Foundation/Foundation.h>
```

Comment:
This can be omitted (even tho its not part of this PR changes)

Reply:
It is needed for NSString, NSError, NSURL, etc